incubator/hnc/cloudbuild.yaml

```diff
@@ -0,0 +1,16 @@
+steps:
+# Build release container image
+- name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/hnc/controller', '.']
+# Build release YAML
+- name: google/cloud-sdk
+  entrypoint: "bash"
+  args:
+  - -c
+  - kubectl kustomize config/default/ > release.yaml
+
+images: ['gcr.io/$PROJECT_ID/hnc/controller']
+artifacts:
+  objects:
+    location: 'gs://artifacts.$PROJECT_ID.appspot.com/hnc/'
```

Comment:
What's appspot.com? I thought that was an AppEngine thing?

Reply:
It's just the a Default Cloud Storage Bucket in the App Engine. It is in the form <project-id>.appspot.com.
Reference
https://cloud.google.com/appengine/docs/standard/python/googlecloudstorageclient/setting-up-cloud-storage

Reply:
Can't we use GCS separately from App Engine?